src/DependencyInjection/SonataNewsExtension.php

```diff
@@ -39,6 +39,7 @@ public function load(array $configs, ContainerBuilder $container)
         $bundles = $container->getParameter('kernel.bundles');
 
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+        $loader->load('actions.xml');
```

Comment:
everything is singular, this one is plural :smile:

Reply:
We already had this discussion:
https://github.com/sonata-project/SonataAdminBundle/pull/5120#discussion_r194295521

Would prefer singular too, but symfony is mainly using plural for this

Reply:
so, we should rename others then

Reply:
Feel free to do this :trollface: